docs/api/server/Environment-Variables.md

```diff
@@ -2,6 +2,461 @@
 
 # Environment Variables
 
-Documentation Forthcoming
+One App can be configured via Environment Variables:
 
-[☝️ Return To Top](#environment-variables)
\ No newline at end of file
+**Contents**
+* [`HOLOCRON_MODULE_MAP_URL`](#holocron_module_map_url)
+* [`HOLOCRON_SERVER_MAX_MODULES_RETRY`](#holocron_server_max_modules_retry)
+* [`HOLOCRON_SERVER_MAX_SIM_MODULES_FETCH`](#holocron_server_max_sim_modules_fetch)
+* [`HTTPS_PORT`](#https_port)
+* [`HTTPS_PRIVATE_KEY_PASS_FILE_PATH`](#https_private_key_pass_file_path)
+* [`HTTPS_PRIVATE_KEY_PATH`](#https_private_key_path)
+* [`HTTPS_PUBLIC_CERT_CHAIN_PATH`](#https_public_cert_chain_path)
+* [`HTTPS_TRUSTED_CA_PATH`](#https_trusted_ca_path)
+* [`HTTP_METRICS_PORT`](#http_metrics_port)
+* [`HTTP_ONE_APP_DEV_CDN_PORT`](#http_one_app_dev_cdn_port)
+* [`HTTP_ONE_APP_DEV_PROXY_SERVER_PORT`](#http_one_app_dev_proxy_server_port)
+* [`HTTP_PORT`](#http_port)
+* [`IP_ADDRESS`](#ip_address)
+* [`NODE_ENV`](#node_env)
+* [`ONE_CLIENT_CDN_URL`](#one_client_cdn_url)
+* [`ONE_CLIENT_LOCALE_FILENAME`](#one_client_locale_filename)
+* [`ONE_CLIENT_REPORTING_URL`](#one_client_reporting_url)
+* [`ONE_CLIENT_ROOT_MODULE_NAME`](#one_client_root_module_name)
+* [`ONE_CONFIG_ENV`](#one_config_env)
+* [`ONE_ENABLE_POST_TO_MODULE_ROUTES`](#one_enable_post_to_module_routes)
+* [`ONE_MAP_POLLING_MAX`](#one_map_polling_max)
+* [`ONE_MAP_POLLING_MIN`](#one_map_polling_min)
+
+## `HOLOCRON_MODULE_MAP_URL`
+
+**Runs In**
+* ✅ Production
+* ✅ Development
+
+URL where the [Module Map] is hosted.
+
+Defaults to `http://localhost:3001/static/module-map.json` if `NODE_ENV` is set to `development` so that [One App Dev CDN] can be leveraged for local development.
```

Comment:
Do you think it would be more clear to readers if the default value was given its own section instead of it being hidden in the description where it may go unnoticed?

Reply:
Updated in b02bcb1